README.md

```diff
@@ -35,13 +35,13 @@ This is an API that [creates AWS (sub)accounts in an Organization](https://docs.
 1. Setup the database with Proof-of-Concept data.
 
     ```sh
-    sqlite3 aws-account-broker.db < poc_data.sql  
+    sqlite3 /tmp/aws-account-broker.db < poc_data.sql  
```

Comment:
I'd keep it local, personally. Fine either way.

Reply:
I wanted to ensure the database files didn't get mixed up with the repo.

Reply:
I’d just add *.db to the gitignore.